### ISSUE TITLE
Changing time and type eligibility displays

### ISSUE THREAD

**Issue by michaelzhang43** [codeforpdx/recordexpungPDX#1190]
Creating an issue here to bookmark this idea:

We currently display a green checkmark for Type eligibility if it is Type eligible, even if the case is not Time-eligible and therefore not Eligible as a whole. I'm thinking that the green checkmark may be misleading. At the same time, the red X would also be inaccurate. 

Let's brainstorm a possible solution in the comments. I will start by proposing that, in the above scenario, we simply omit the green check

### This task is done when:
 -
 -

**Comment by wittejm** [comment]
Getting rid of the green check mark sounds like a good change.

Another thought on this is that "Type Eligibility" has become more complicated and difficult to express succinctly. The 22 (!) types are defined by a patchwork of their identifying information, relevant statute subsections, and case law / convention. 

Also, "a charge must be both time eligible and type eligible" itself is getting messy because these two criteria aren't fully distinct. Class B felony is the best example; also the <1 oz Marijuana charge, and MarijuanaUnder21 have type-based time eligibility. 

The key information the user needs to see is the answer to "This charge is eligible because __" . Our type eligibility Reason text covers most of this. Could we go further and hide charge type names, and only show reason text, And re-word the reason text where necessary?

**Comment by michaelzhang43** [comment]
Thanks for pinging on this, Hunter - I had lost track of this issue

Jordan - I agree that Type and Time eligibility concepts overlap and are potentially confusing categories esp. for the Chargetypes you named. 

I'm inclined to think that we can do away with the icons altogether, as Hunter marked up in the No Icons panel. 

What do yall think @wittejm @KentShikama

**Comment by KentShikama** [comment]
@michaelzhang43 I think that sounds good as a short term solution.

As a longer term solution, I'm wondering whether we could do something like the network diagnostics tool on OS X.

![Screen Shot 2020-06-11 at 7 07 11 AM](https://user-images.githubusercontent.com/6329898/84394845-4ea22900-abb2-11ea-88a3-f038f6d71808.png)

To make it even clearer that each step is linked to the next one, there should be a "river" (line with an arrow pointing in the direction the "river" flows) that connects each step. As @wittejm mentioned, for some charges like MJ <1, the time eligibility function (step) is just the identity function, but I think it is fine to just state the fact that it is an identity function.

Our version would have four nodes: type eligibility, time eligibility, balance owed, and final verdict. The color of the final verdict node should match the top-level eligibility label.

When there are `n` ambiguities, the river should split into `n` individual rivers at the top and merge at the bottom for the final verdict.

**Comment by michaelzhang43** [comment]
what? @KentShikama

**Comment by KentShikama** [comment]
@michaelzhang43 Hmm? Did you want to call?

**Comment by hmarcks** [comment]
I see what you're saying @KentShikama , isn't it very similar to the current approach, except we aren't including Balance? At least this was how I was thinking about it when designing it. To make it more clear each node would be more visually connected.

**Comment by KentShikama** [comment]
Indeed. I guess I'm imaging the river flows downwards instead of upwards like it currently is.

**Comment by KentShikama** [comment]
Linking https://github.com/codeforpdx/recordexpungPDX/pull/1303

**Comment by KentShikama** [comment]
@michaelzhang43 I believe this issue can be closed now?

**Comment by hmarcks** [comment]
One thing that hasn't been updated yet is the Time icon when ineligible:
![Screen Shot 2020-07-26 at 10 47 29 PM](https://user-images.githubusercontent.com/6887982/88501796-143df100-cf92-11ea-8682-206969bee410.png)
